js/app.js

```diff
@@ -0,0 +1,66 @@
+const input = document.querySelector(".contact__input");
+const sendBtn = document.getElementById("contact__submit--send");
+const closeBtn = document.getElementById("contact__submit--close");
+const modal = document.querySelector(".addressbook");
+const emails = [];
+
+const addEmail = (e, tab) =>{
+  e.preventDefault();
+  tab.push(input.value);
+  tab.reverse();
+  console.log(tab);
```

Comment:
Jeśli udostępniasz, usuwaj debugi i console.log'i. Mnie to osobiście drażni. Chyba że jest ku temu jakiś powód

Reply:
ok, będę pamiętał.